app/main/views/frameworks.py

```diff
@@ -251,7 +252,10 @@ def get_framework_suppliers(framework_slug):
         )
 
     return jsonify(supplierFrameworks=[
-        supplier_framework.serialize(with_users=False) for supplier_framework in supplier_frameworks
+        supplier_framework.serialize(
+            with_users=False,
+            with_declaration=convert_to_boolean(request.args.get("with_declarations", "true")),
```

Comment:
Would it be worth pulling the `convert_to_boolean` out before this so it doesn't get called for every supplier_framework?